agent/rpm/pbench-agent.spec.j2

```diff
@@ -8,16 +8,22 @@ URL:            https://github.com/distributed-system-analysis/pbench
 Source0:        pbench-agent-%{version}.tar.gz
 Buildarch:      noarch
 
-%if 0%{?rhel} < 9
-Requires:  ansible
-%else
+# EPEL provides ansible (a curated set of roles with a dependency on ansible-core)
+# on RHEL 8 and RHEL9. Only CentOS-Stream seems to not have an ansible package
+# available.
+
+%if 0%{?centos} >= 8
 Requires:  ansible-core
+%else
+Requires:  ansible
 %endif
 
-%if 0%{?rhel} == 7
+
+%if 0%{?rhel} == 7 
```

Comment:
There's a trailing space on this line (dunno whether we should care...but that's what makes it show up in the diffs).

Reply:
I think it's mostly the extra empty line that I added: I was trying to separate sections more visibly.  I'll get rid of the trailing space.